examples/Example120_ThreeRegions1D.jl

```diff
@@ -22,34 +22,38 @@ function reaction(f, u, node, data)
     else
         f[1] = 0
     end
+    return nothing
 end
 
 function source(f, node, data)
     if node.region == 1
         f[1] = 1.0e-4 * (3.0 - node[1])
     end
+    return nothing
 end
 
-## Since 0.17.0 one can 
+## Since 0.17.0 one can
 ## write into the result also where
 ## the corresponding species has not been enabled
 ## Species information is used to prevent the assembly.
 function correctionflux(f, u, edge, data)
```

Comment:
```suggestion
function correctionflux!(f, u, edge, data)
```
These should all be marked as 'mutating the parameters' right?

Reply:
Basically all of them. But it is just names, no difference in semantics. However, there is an inconsistency: I can have "!" in function names, but not in struct members or keyword arguments.
I would keep this as it is in the moment. May be the Azubis could help out here.

Reply:
In some examples, e.g. 160, `flux`, `reaction` etc. are already marked with a bang. 
I think we should use this formatting point in time to remove the inconsistency as well.